editors/kibi/Portfile

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github  1.0
+PortGroup           cargo   1.0
+
+github.setup        ilai-deutel kibi 0.3.0 v
+github.tarball_from archive
+revision            0
+
+description         Text editor in ≤1024 lines of code, written in Rust
+
+long_description    A configurable text editor with UTF-8 support, \
+                    incremental search, syntax highlighting, line numbers and more, \
+                    written in less than 1024 lines1 of Rust with minimal dependencies.
+
+categories          editors
+installs_libs       no
+license             Apache-2 MIT
+maintainers         {@sikmir disroot.org:sikmir} \
+                    openmaintainer
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  ea11b171e5d25ac1cd462a6531375df1692e8701 \
+                    sha256  086eeb4c9ffaae98c02c39d932796987590978b5907ed3e6ac5d44aeabec176c \
+                    size    1235967
+
+build.pre_args-delete \
```

Comment:
I think these days the preference is to set `cargo.offline_cmd` to an empty value.

Example:
```
...
checksums           ${distname}${extract.suffix} \
                    rmd160  ea11b171e5d25ac1cd462a6531375df1692e8701 \
                    sha256  086eeb4c9ffaae98c02c39d932796987590978b5907ed3e6ac5d44aeabec176c \
                    size    1235967

cargo.offline_cmd


post-patch { ...
```